cypress/e2e/tests/pages/charts/logging.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { ChartPage } from '@/cypress/e2e/po/pages/explorer/charts/chart.po';
+import HomePagePo from '@/cypress/e2e/po/pages/home.po';
+import { InstallChartPage } from '@/cypress/e2e/po/pages/explorer/charts/install-charts.po';
+import ProductNavPo from '@/cypress/e2e/po/side-bars/product-side-nav.po';
+import { LoggingClusteroutputEditPagePo, LoggingClusteroutputListPagePo } from '@/cypress/e2e/po/other-products/logging/logging-clusteroutput.po';
+import { LoggingClusterflowEditPagePo, LoggingClusterflowListPagePo } from '@/cypress/e2e/po/other-products/logging/logging-clusterflow-po';
+import Kubectl from '@/cypress/e2e/po/components/kubectl.po';
+
+describe('Logging Chart', { testIsolation: 'off', tags: ['@charts', '@adminUser'] }, () => {
+  before(() => {
+    cy.login();
+    HomePagePo.goTo();
+  });
+
+  it('is installed and a rule created', () => {
+    cy.updateNamespaceFilter('local', 'none', '{"local":[]}');
```

Comment:
lets update this in the afterAll hook to the default value (`Only User Namespaces`) in case other tests expect it

Reply:
I'm going to change it, but those other tests should set their own state before execution.